Sources/PointFree/Database.swift

```diff
@@ -5,6 +5,7 @@ import PostgreSQL
 
 public struct Database {
   var addUserIdToSubscriptionId: (User.Id, Subscription.Id) -> EitherIO<Error, Prelude.Unit>
+  var createFeedRequestEvent: (FeedRequestEvent.FeedType, String, User.Id) -> EitherIO<Error, Prelude.Unit>
```

Comment:
this is a new database endpoint for creating a `FeedRequestEvent` which basically just tracks all user agents from requests to the private RSS endpoint.